src/app/personside/infotabs/meldinger/traadvisning/NyEnkeltmelding.tsx

```diff
@@ -0,0 +1,160 @@
+import * as React from 'react';
+import { useRef } from 'react';
+import { LestStatus, Melding } from '../../../../../models/meldinger/meldinger';
+import Snakkeboble from 'nav-frontend-snakkeboble';
+import { erChatMelding, erMeldingFraBruker, erMeldingFraNav, meldingstittel } from '../utils/meldingerUtils';
+import { formatterDatoTid } from '../../../../../utils/date-utils';
+import styled from 'styled-components/macro';
+import Tekstomrade, { createDynamicHighlightingRule, defaultRules, Rule } from 'nav-frontend-tekstomrade';
+import { rule as sladdRule } from '../../../../../utils/sladdrule';
+import Etikett from 'nav-frontend-etiketter';
+import { guid } from 'nav-frontend-js-utils';
+import theme from '../../../../../styles/personOversiktTheme';
+import './enkeltmelding.less';
+import { SpaceBetween } from '../../../../../components/common-styled-components';
+
+interface Props {
+    melding: Melding;
+    sokeord: string;
+    meldingsNummer: number;
+}
+
+const StyledLi = styled.li`
+    .snakkeboble-panel {
+        flex-basis: 40rem;
+        padding: 0.5rem 0.5rem 0.3rem;
+    }
+    a {
+        word-break: break-word; /* Hvis ikke kan snakkeboblen vokse seg ut av container uten mulighet for scroll */
+    }
+    .snakkeboble {
+        margin: 0;
+    }
+    @media print {
+        page-break-inside: avoid;
+        border: ${theme.border.skille};
+        margin-bottom: 2rem;
+
+        .snakkeboble__snakkeboble-pil-container,
+        .nav-ikon,
+        .bruker-ikon {
+            display: none;
+        }
+
+        .snakkeboble-panel {
+            flex-basis: 100%;
+        }
+    }
+`;
+
+const BoldTekstomrade = styled(Tekstomrade)`
+    font-weight: 600;
+`;
+const Bunntekst = styled.div`
+    /* Trengs for borders */
+`;
+
+const MeldingInfoWrapper = styled.div`
+    display: flex;
+    flex-direction: row;
+    gap: 10px;
+    align-items: left;
+    background: white;
+`;
```

Comment:
Kan forenkles til bare to properties uten at det påvirker det visuelle.

```suggestion
const MeldingInfoWrapper = styled.div`
    display: flex;
    gap: 0.5rem;
`;
```